### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by estherkim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
 Facilitator: @rsimha(https://github.com/rsimha)
 
 # Members
-- [@ajwhatson](https://github.com/ajwhatson) - Anna Watson
```
Comment (rcebulko) [inline README.md:7]: This makes me wonder if we should add a "Past contributors" section to give attribution to Anna, Erwin, Marcial, etc.

**Review by rcebulko — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rsimha) [reply README.md:7]: Great idea. UI does this: https://github.com/ampproject/wg-ui-and-a11y#past-members

**Review by rsimha — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (estherkim) [reply README.md:7]: +1 great idea. Added.

**Review by Rafer45 — APPROVED** [review]
LGTM!

**Pull request merged**